src/admin.js

```diff
@@ -12,7 +12,6 @@ domReady(() => {
 	Object.keys(GatherPress.unregister_blocks).forEach((key) => {
 		// eslint-disable-next-line no-undef
 		const blockName = GatherPress.unregister_blocks[key];
-
```

Comment:
not sure why this file was touched. let's revert. spacing out code is easier for people to read and work with.